keras/layers/core/einsum_dense_test.py

```diff
@@ -496,3 +500,63 @@ def test_quantize_when_already_quantized(self):
             ValueError, "`quantize` can only be done once per layer."
         ):
             layer.quantize("int8")
+
+    @pytest.mark.requires_trainable_backend
+    def test_quantize_when_lora_enabled(self):
```

Comment:
With the TF backend, please add a `model.export()` test (check exporting a TFSavedModel and reloading it via `TFSMLayer`).

Reply:
Added